OrbitGl/GraphTrack.cpp

```diff
@@ -32,15 +33,32 @@ void GraphTrack::Draw(GlCanvas* canvas, PickingMode picking_mode) {
   }
 
   float track_z = GlCanvas::kZValueTrack;
-  float text_z = GlCanvas::kZValueText;
 
   Box box(pos_, Vec2(size_[0], -size_[1]), track_z);
 
   // Draw label with graph value at current mouse position.
-  const Color kLabelTextColor(255, 255, 255, 255);
+  const Color kWhiteColor(255, 255, 255, 255);
+  const Color kBlackColor(0, 0, 0, 255);
   double graph_value = GetValueAtTime(time_graph_->GetCurrentMouseTimeNs());
-  canvas->GetTextRenderer().AddText(std::to_string(graph_value).c_str(), canvas->GetMouseX(), y0,
-                                    text_z, kLabelTextColor);
+
+  int white_text_box_width;
```

Comment:
Maybe initialize with default value?

Reply:
With 0 is ok?

Reply:
See my other suggestion to just return the width from GetStringWidth, then you don't have to initialize with a default value here.